distribution/docs/src/main/resources/content/_sources/solr-catalog-provider.adoc

```diff
@@ -75,12 +68,6 @@ Zookeeper helps manage the overall structure.
  +------------------+   +---------------------+
 ....
 
-[NOTE]
-====
-Solr Cloud is currently in Beta version.
-Bugs and feature changes may affect the final release.
-====
-
```

Comment:
It was not a good idea to document the settings both here and in global-setting.adoc. Removed.